extensions/interactions/CodeRepl/CodeRepl.py

```diff
@@ -23,8 +23,12 @@ class CodeRepl(base.BaseInteraction):
     name = 'Code Editor'
     description = 'Allows learners to enter code and get it evaluated.'
     display_mode = base.DISPLAY_MODE_SUPPLEMENTAL
-    is_trainable = False
-    _dependency_ids = ['skulpt', 'codemirror']
+    is_trainable = True
```

Comment:
@seanlip @anmolshkl why do we have 2 constants for trainable classifiers? I did some digging and found that is_trainable is used in trainaing interface of editor view (where one creates training data) and `is_interaction_trainable` is used by classifier jobs and answer classification service part in learner view. Can we merge them into a single constant?